en/frames_multicopter/holybro_x500v2_pixhawk6c.md

```diff
@@ -0,0 +1,159 @@
+# Holybro X500 V2 + Pixhawk 6C (PX4 Dev Kit)
```

Comment:
We can't just call it the PX4 Dev Kit - this is already the second "Devkit" produced by Holybro, and they name it the "Holybro X500 V2 ARF Kit" at their link.